__tests__/app.test.js

```diff
@@ -14,6 +15,14 @@ afterAll(() => {
 
 describe("app", () => {
   describe("/api", () => {
+    test("GET /api should return with an object with keys that show all the available endpoints on the API", () => {
+      return request(app)
+        .get("/api/")
+        .expect(200)
+        .then((res) => {
+          expect(res.body).toEqual(data);
+        });
+    });
```

Comment:
Good test. I'd encourage you to add the expected status code in the test description (200).